src/main/java/com/rolfje/anonimatron/Anonimatron.java

```diff
@@ -76,9 +91,18 @@ public static void main(String[] args) throws Exception {
 		}
 	}
 
-	private static void anonymize(String configFile, String synonymFile, boolean dryrun)
+	private static void anonymize(String jdbcurl, String userid, String password, String configFile, String synonymFile, boolean dryrun)
 			throws Exception, SQLException {
 		Configuration config = Configuration.readFromFile(configFile);
+		if (jdbcurl != null) {
+			config.setJdbcurl(jdbcurl);
+		}
+		if (userid != null) {
+			config.setUserid(userid);
+		}
+		if (password != null) {
+			config.setPassword(password);
+		}
```

Comment:
Neat! That would also enable users to create a config file without having to have a password in it. Overriding just the password on the command line. Thanks!